lib/OctokitAction.ts

```diff
@@ -108,7 +102,7 @@ export abstract class OctokitAction extends Action {
                   }
               }
           }`);
-    return nodes;
+    return data?.organization?.samlIdentityProvider?.externalIdentities?.nodes ?? [];
```

Comment:
With destructing, if any property is null, we can get an error like:
```
TypeError: Cannot read properties of null (reading 'externalIdentities')
```
Which will stop the execution of the action.

Reply:
It's probably better to let it fail hard. Otherwise token issues would be hidden deep under indirect indices